### ISSUE TITLE
Zoom out function does not always work as expected

### ISSUE THREAD

**Issue by lparsons** [ElucidataInc/ElMaven#588]
I'm not sure if this is related to fixes for #553 (e.g. #587, #580, #584), but the zoom out button does not seem to work reliably for me anymore. See the following screenshots as an example.

Look at `C14:0` with *Auto Zoom* enabled, everything looks as expected:
![image](https://user-images.githubusercontent.com/645128/35872070-db8faca6-0b33-11e8-93fb-59d99da4e305.png)

Click `Betaine Aldehyde` with no RT in the database, RT line disappears, but zoom is now from 6.00 to 18.00 and nothing is displayed.
![image](https://user-images.githubusercontent.com/645128/35872150-14ea9498-0b34-11e8-9685-d62eaad1b058.png)

Click *Zoom Out* and nothing happens, zoom level doesn't change:
![image](https://user-images.githubusercontent.com/645128/35872193-3363f874-0b34-11e8-979a-89c870eb0ca4.png)

Click *ATP* (which has an RT) and window is zoomed to expected location (`RT==14.94`). However, no data is displayed (not sure if there is data or not)
![image](https://user-images.githubusercontent.com/645128/35872326-8bcdadac-0b34-11e8-8775-6c38dd3a0687.png)

Clicking *Zoom Out* both **erases the RT line** and **zooms to 6.00-18.00** again (which is not expected)
![image](https://user-images.githubusercontent.com/645128/35872344-94364ada-0b34-11e8-99ce-95fd3c11c323.png)

**Comment by shubhra-agrawal** [comment]
@lparsons This is similar to the corner case I mentioned in the comments for #553. I have raised a PR for a fix for this already. It will be merged asap once we have tested it to make sure no similar bugs arise as a result. This will be available in v0.2.4.

Just to clarify, when nothing is displayed it's because there is no data there and the zoomed out Rt range is not random but the range of the whole sample. I've changed it to display "EMPTY EIC" in such cases and zoom out to the total Rt range of a sample. Also, the case of missing title+empty EIC window for MS2 data with no expected Rt has been handled.
